package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Utilities for Node.js core collaborators",
   "type": "module",
   "engines": {
-    "node": ">=16.8.0"
+    "node": ">=18.0.0"
```

Comment:
```suggestion
    "node": "^18.18.0 || >=20.0.0"
```

Reply:
why the 20 is needed?

Reply:
20.x is the current release line, it makes sense to support it. My suggestion is to not use a range that includes 19.x (notice the `^` before `18.18`).

Reply:
oh, I forgot 19 is dead